src/jimgw/core/single_event/likelihood.py

```
@@ -555,9 +561,11 @@ class HeterodynedTransientLikelihoodFD(SingleEventLikelihood):
     n_bins: int
     reference_parameters: dict
     freq_grid_low: Array
+    freq_grid_high: Array
     freq_grid_center: Array
+    bin_widths: Array
     waveform_low_ref: dict[str, Float[Array, " n_bin"]]
```

Ruff (0.15.14)

[error] Syntax error in forward annotation: Expected an expression

(F722)